packages/tx/src/util.ts

```diff
@@ -116,6 +129,98 @@ export class AccessLists {
   }
 }
 
+export class AuthorizationLists {
+  public static getAuthorizationListData(
+    authorizationList: AuthorizationListBytes | AuthorizationList
+  ) {
+    let AuthorizationListJSON
+    let bufferAuthorizationList
+    if (isAuthorizationList(authorizationList)) {
+      AuthorizationListJSON = authorizationList
+      const newAuthorizationList: AuthorizationListBytes = []
+      const jsonItems = ['chainId', 'address', 'nonce', 'yParity', 'r', 's']
```

Comment:
would just reading the properities and comparing them to undefined with a `||` be simpler ?

Reply:
I'm not sure what you mean here?